controls/mousestates/cMouseNormalState.cpp

```diff
@@ -1,8 +1,25 @@
 #include <algorithm>
-#include "d2tmh.h"
+
+#include "controls/cGameControlsContext.h"
+#include "controls/cMouse.h"
+
+#include "utils/cRectangle.h"
+#include "gameobjects/units/cUnit.h"
+#include "gameobjects/structures/cAbstractStructure.h"
+#include "player/cPlayer.h"
+#include "sidebar/cSideBar.h"
+
+#include "cGame.h"
+
+// instead of using d2tmc.h
+extern cGame		  game;
+extern cUnit          unit[MAX_UNITS];
```

Comment:
🤔 Although it saves including d2tmc.h, I wonder if it's worth it. If we ever clean up that file (by moving data into the game class, for instance), then we need to touch these lines too and we may forget. I hope we get linker errors by then, but it makes me nervous to declare variables in multiple places. :)